### DISCUSSION TITLE
Images::public on image path in antler template

### DISCUSSION THREAD

**Discussion by chiptu** [statamic/cms#6004] [Q&A]
Hey everyone,

When i try to display my image inside my antler i get images::public/example.jpeg .

In my collection i have only one image uploaded, i have a public asset container.

The article collection data is found in my template and when i dump my image_de_presentation data it seems correct.
![Capture d’écran 2022-05-05 à 23 26 39](https://user-images.githubusercontent.com/35498845/167028616-156885f6-0071-431b-a52e-37531b7c7599.png)
![Capture d’écran 2022-05-05 à 23 26 23](https://user-images.githubusercontent.com/35498845/167028624-267ae43d-a407-4592-ae40-3552fe2562d1.png)
![Capture d’écran 2022-05-05 à 23 29 20](https://user-images.githubusercontent.com/35498845/167028720-9be682a3-262d-4f8c-a54a-aed1c961b067.png)

What am i missing out ?

**Comment by edalzell** [comment]
Let's see the yaml data for that article and the corresponding blueprint.

**Reply by chiptu** [reply]
![Capture d’écran 2022-05-06 à 09 51 33](https://user-images.githubusercontent.com/35498845/167097199-c59b3071-9092-4415-bebe-83af87ff4d83.png)
![Capture d’écran 2022-05-06 à 09 52 58](https://user-images.githubusercontent.com/35498845/167097223-07400f50-0dad-49ae-99f9-23760f4512b5.png)
![Capture d’écran 2022-05-06 à 09 53 13](https://user-images.githubusercontent.com/35498845/167097254-4d3e281a-a794-4f8a-a935-934a380077e0.png)
![Capture d’écran 2022-05-06 à 09 53 28](https://user-images.githubusercontent.com/35498845/167097283-73429a2e-7179-42bb-a8a3-fe8c93717406.png)
![Capture d’écran 2022-05-06 à 09 53 34](https://user-images.githubusercontent.com/35498845/167097293-a1f02cea-6516-4590-b35d-ef9af2dea2f1.png)

Thanks for your quick response (:  

Here's the yaml file, the blueprint, the asset detail , the container and the folder with the images.

**Reply by duncanmcclean** [reply]
I think @edalzell also wanted to see the entry's markdown file.

**Reply by chiptu** [reply]
Oh okay i'm totally new to statamic i didn't get it.

Here it is an example of article collection markdown

```
---
id: ccd025f2-835c-46bc-9beb-88ef59ce9543
blueprint: article
title: '🤔 QrCodes Quésaco ?'
image_de_presentation:
  - public/exemple.jpg
lien_linkedin: 'https://www.linkedin.com'
temps_lecture: 2
resume: |-
  On en voit partout depuis le COVID-19 mais au juste d'où viennent ces carrés étranges ? <br>
  🕵️  Espionnage étatique ? 👽  Alien ? ou pire encore conspiration de geeks 🤓 ?<br>
  Pourquoi on en voit partout alors qu'il y a 2 ans on en voyait aucun  ? <br>
  On va essayer de clarifier tout ça !
updated_by: 1
updated_at: 1651784746
---
Test contenu
```

**Reply by chiptu** [reply]
I verified i have image magick and gd library on my macos , i m using statamic with a local laravel 9 installation and php 8.1 .

Should i move it to issue ?
